### PR TITLE
infra: remove cache from drone execution

### PR DESCRIPTION
**Pull request by nick-mancuso**
"Fixes" (really allows us to bypass) failure noted at: https://cloud.drone.io/checkstyle/checkstyle/11222/4/2

I am guessing that this is the same issue mentioned by @romani regarding dropbox last week in discord:

```
Dropbox.
Hi Roman,
This email is an automated notification from Dropbox letting you know that your Public links have been temporarily suspended for generating excessive traffic.

```
The file we are actually downloading now:

```
➜  checkstyle git:(fix-infra-2) ✗ curl -o .ci-temp/cache.tar -SsL \
    https://www.dropbox.com/s/"sh1g8o4h16p4n""x6"/cache.tar\?dl\=0

➜  checkstyle git:(fix-infra-2) cat .ci-temp/cache.tar 
<!DOCTYPE html>
<html>
<head><meta http-equiv="Content-Type" content="text/html; charset=utf-8">
<title>Dropbox - 404</title>
<link href="https://cfl.dropboxstatic.com/static/css/error.css" rel="stylesheet" type="text/css"/>
<link rel="shortcut icon" href="https://cfl.dropboxstatic.com/static/images/favicon.ico"/>
</head>
<body>
<div class="figure">
<img src="https://assets.dropbox.com/www/en-us/illustrations/spot/look-magnifying-glass.svg" alt="Error: 404"/>
</div>
<div id="errorbox">
<div class="not-found">
                <h1>Error (404)</h1>
                We can't find the page you're looking for.
                <div class="not-found--links">
                    Here are a few links that may be helpful:
                    <ul>
                        <li><a href="https://www.dropbox.com/home?_tk=fof">Home</a></li>
                        <li><a href="https://www.dropbox.com/help?_tk=fof">Help center</a></li>
                        <li><a href="https://www.dropbox.com/login?_tk=fof">Sign in</a></li>
                        <li><a href="https://www.dropbox.com/register?_tk=fof">Get a free account</a></li>
                        <li><a href="https://www.dropbox.com/plus?_tk=fof">Dropbox Plus</a></li>
                        <li><a href="https://www.dropbox.com/business?_tk=fof">Dropbox Business</a></li>
                    </ul>
                </div>
            </div>
</div>

</body>
</html>


```

### CODE REVIEW

**Comment by romani** [comment]
This is not required to merge, as side effect that we have in drone it not blocking execution.

But it shows very strange situation that I can not understand, there is no m2 cache installation but there is no download of artifacts.

If we get understanding of this paradox, I would be ok to remove restoration of cache at all.

**Comment by nick-mancuso** [comment]
Link to build: https://cloud.drone.io/checkstyle/checkstyle/11289/1

**Comment by nick-mancuso** [comment]
@romani see https://cloud.drone.io/checkstyle/checkstyle/11289/1/2, we do not have cache, but download of all dependencies is very fast.

File: .ci/validation.sh [hunk 1]
```diff
@@ -381,7 +381,7 @@ verify-regexp-id)
 
 checkstyle-and-sevntu)
   export MAVEN_OPTS='-Xmx2000m'
-  mvn -e --no-transfer-progress clean verify -DskipTests -DskipITs --no-transfer-progress \
+  mvn -e --no-transfer-progress clean verify -DskipTests -DskipITs \
```
Comment (romani) [inline .ci/validation.sh:384]: Keep it, it is good option

**Review by romani — CHANGES_REQUESTED** [review]
Items

File: .ci/validation.sh (hunk 1 above)
Comment (nick-mancuso) [reply .ci/validation.sh:384]: It is a duplicate

**Comment by nick-mancuso** [comment]
Link to drone execution in this PR (no cache):

image

Link to drone execution from another PR:

**Comment by romani** [comment]
Please put in commit reference to this PR and merge PR immediately

**Review by romani — APPROVED** [review]
Ok to merge

**Comment by nick-mancuso** [comment]
> Please put in commit reference to this PR and merge PR immediately

@romani done

**Pull request merged**